tools/eslint-plugin/dist/configs/extension-exports-config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
@johnnyd710 Can you add the copyright header? Not sure why it was missed here by the commit hook.